src/main/resources/emissary/output/DropOffUtil.cfg

```diff
@@ -16,46 +16,49 @@ DATE_PARAMETER = "EventDate"
 DATE_PARAMETER = "FILE_DATE"
 
 # see SimpleDateFormat javadocs to understand these
```

Comment:
Is this comment still accurate? Should it be updated to now reference `DateTimeFormatter` instead of `SimpleDateForma`?

Reply:
I thought I changed that, definitely should be updated.